src/lib/api/supabase/projects/api.ts

```diff
@@ -77,6 +77,8 @@ export async function queryDBProjects(
   if (opts.archived) query = query.is('archived', true)
   else query = query.not('archived', 'is', true)
   if (opts.pv?.length) query = query.in('pv', opts.pv)
+  if (opts.owner) query = query.ilike('owner', opts.owner)
+  if (opts.creator) query = query.ilike('creator', opts.creator)
```

Comment:
the /projects api endpoint now supports filtering projects by `owner` and `creator`